### PR TITLE
feat(frontend): run nginx as non-root user + fix prod build

### PR DESCRIPTION
**Pull request by DavidSpek**
## Summary
Runs the frontend container as non-root.

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: www/nginx.conf [hunk 1]
```diff
@@ -1,48 +1,37 @@
 server {
-    listen       80;
-    server_name  localhost;
-    #charset koi8-r;
-    #access_log  /var/log/nginx/host.access.log  main;
+    # listen on port 80
+    listen 8080;
```
Comment (floreks) [inline www/nginx.conf:3]: Should it be `listen 80;`?

Comment (DavidSpek) [reply www/nginx.conf:3]: 80 is a privileged port and requires root (same for 443). Since this uses the non-privileged NGINX it listens on port 8080.

Comment (floreks) [reply www/nginx.conf:3]: So it is just the comment

Comment (DavidSpek) [reply www/nginx.conf:3]: yep

**Pull request merged**